subcommands/targets/offline-update.go

```diff
@@ -163,6 +175,59 @@ func getTargetInfo(factory string, targetName string) (*ouTargetInfo, error) {
 	return &info, nil
 }
 
+func getWaveTargetMeta(factory string, targetName string, wave string) (*tuf.FileMeta, error) {
+	waveTargets, err := api.WaveTargetsList(factory, true, wave)
+	if err != nil {
+		if herr := client.AsHttpError(err); herr != nil && herr.Response.StatusCode == 404 {
+			return nil, fmt.Errorf("no active Wave with the specified name was found; wave: %s", wave)
```

Comment:
@doanac was commenting on this, but there is no (convenient) way to reply to that comment.

> I see you are moving towards doing error messages with a lowercase first word.
> This makes the Golang linters happier, but its not how we've traditionally done our other stuff.
> I think this is probably fine - we need to stop the bleeding, I suppose. But lets make a conscious decision that this is what we want to do.

So, I have an opinion on this.

I have no strong preference for Hungarian versus lowercase notation in error messages.
One use case where the Hungarian wins is when we have multi-sentence errors, which are rather common for the CLI (fioctl in particular).

IIUC we are using Hungarian-style (sentence-like) errors in the majority of places across Fioctl (yes, with first capital letter).
So, in order to look more professional, I'd recommend to continue using that way universally, until we decide to make a global change.
A third-party Golang linter is the last thing I care about when making such decisions.

Reply:
So, we have an consensus on the first capital letter? @doanac

Reply:
Let's stay with Capital letters and punt this decision for now